static/public/index.html

```diff
@@ -4245,3599 +7239,2764 @@ <h3 class="text-base font-medium text-dark">25%</h3>
-                  </label>
-                  <svg
-                    class="mr-xs"
-                    width="16"
-                    height="16"
-                    viewBox="0 0 16 16"
-                    fill="none"
-                    xmlns="http://www.w3.org/2000/svg"
-                  >
-                    <path
-                      d="M4.99951 7.33334C5.5518 7.33334 5.99951 6.88562 5.99951 6.33334C5.99951 5.78105 5.5518 5.33334 4.99951 5.33334C4.44723 5.33334 3.99951 5.78105 3.99951 6.33334C3.99951 6.88562 4.44723 7.33334 4.99951 7.33334Z"
-                      fill="#5B6366"
-                    />
-                    <path
-                      d="M6.99951 9.33333L5.99951 8L3.99951 10.6667H5.99951H7.99951H11.9995L8.99951 6.66666L6.99951 9.33333Z"
-                      fill="#5B6366"
-                    />
-                    <path
-                      d="M13.3325 2.66666H2.66585C1.93052 2.66666 1.33252 3.26466 1.33252 4V12C1.33252 12.7353 1.93052 13.3333 2.66585 13.3333H13.3325C14.0679 13.3333 14.6659 12.7353 14.6659 12V4C14.6659 3.26466 14.0679 2.66666 13.3325 2.66666ZM2.66585 12V4H13.3325L13.3339 12H2.66585Z"
-                      fill="#5B6366"
-                    />
-                  </svg>
-                  <span
-                    class="text-sm hidden md:inline-block font-medium text-dark-grey"
-                    >Image Poll</span
-                  >
-                </div>
-                <div
-                  class="cursor-pointer misc-box mr-m flex items-center bg-grey-bg py-xxsv px-sv rounded-full"
-                >
-                  <label class="radio-cat mr-xs">
-                    <input type="radio" class="radio" name="survey-image" />
-                    <div class="radio-box radio-small radio-tab"></div>
-                  </label>
-                  <svg
-                    class="mr-xs"
-                    width="16"
-                    height="16"
-                    viewBox="0 0 16 16"
-                    fill="none"
-                    xmlns="http://www.w3.org/2000/svg"
-                  >
-                    <path
-                      d="M4.6665 7.33333H9.33317V8.66666H4.6665V7.33333ZM4.6665 4.66666H11.9798V6H4.6665V4.66666ZM4.6665 10H13.3332V11.3333H4.6665V10ZM2.6665 2.66666H3.99984V13.3333H2.6665V2.66666Z"
-                      fill="#5B6366"
-                    />
-                  </svg>
-                  <span
-                    class="text-sm hidden md:inline-block font-medium text-dark-grey"
-                    >Text Poll</span
-                  >
+
+
+            <div class="flex flex-row justify-between mb-10">
+                <div class="flex flex-col">
+                <div>
+                    <label class="block font-light text-dark-grey text-xs ">Label</label>
+                    <input type="text" placeholder="Enter text" class=" placeholder-grey-shd1 w-72 h-10 p-3 rounded-md border-2 border-solid border-grey-shd5">
```

Comment:
1- border-width should be 1px instead of 2px
2- inputs should have a custom border-width and radius when focused
3- placeholder should be dark when focused as well